public/html/dashboard.html

```diff
@@ -4,20 +4,20 @@
 		<meta charset="UTF-8">
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-		<title>Login Page</title>
+		<title>Dashboard</title>
 
-    <link rel="stylesheet" href="../css/main.css">
-    <script src="../js/main.js" defer></script>
+    <link rel="stylesheet" type="text/css" href="../css/main.css">
+    <link rel="stylesheet" type="text/css" href="../css/dashboard.css">
```

Comment:
Hier können absolute Pfade verwendet werden (`/css/main.css`)

Reply:
Can or should? Is it better to you absolute paths here?

Reply:
You should it makes the path independant from the browser path.
So you can use the same scr regardless, if the brower path is /dashboard or /profiles/231/details